packages/dashboard/src/components/dashboard/main-menu.tsx

```diff
@@ -1,29 +1,64 @@
-import { Divider, List, ListItem, makeStyles, Typography } from '@material-ui/core';
+import { Divider, makeStyles, Typography, Button } from '@material-ui/core';
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
 import Debug from 'debug';
 import React from 'react';
 import DashboardTooltip from 'react-components/lib/tooltip';
 import { TooltipsContext } from '../app-contexts';
 import { OmniPanelViewIndex } from './dashboard';
+import { MainMenuItemState } from './main-menu-item-state';
```

Comment:
`main-menu-item-state` depends on `main-menu` as well. Try to avoid recursive dependencies whenever possible, I didn't check but this may also apply to `main-menu-task-state` etc.